src/scicookie/ui.py

```diff
@@ -1,19 +1,26 @@
 """Define functions for the interface with the user."""
 from __future__ import annotations
 
-from typing import Optional, Type
+import os
+
+from typing import Any, Optional, Type
 
 import inquirer
+
+from colorama import Fore, Style, init
 from jinja2 import Template
 
 from scicookie.logs import SciCookieErrorType, SciCookieLogs
 
+# Initialize Colorama
+init()
+
 
 def _create_question(
     question_id: str, question: dict
 ) -> Optional[inquirer.questions.Question]:
```

Comment:
The types in the codebase can be simplified, for example -
```suggestion
) -> inquirer.questions.Question | None:
```
But thus is not related to this PR. I can create a follow-up PR to update the typing hints.

Reply:
nice! it would be great to have a follow-up pr for that :) thank you